Cargo.toml

```diff
@@ -104,14 +104,14 @@ serde_cbor = "0.9.0"
 serde_derive = "1.0.27"
 serde_json = "1.0.10"
 socket2 = "0.3.8"
-solana_program_interface = { path = "common", version = "0.10.0" }
+solana-sdk = { path = "common", version = "0.10.0" }
 sys-info = "0.5.6"
 tokio = "0.1"
 tokio-codec = "0.1"
 untrusted = "0.6.2"
-noop = { path = "programs/native/noop" }
-bpf_loader = { path = "programs/native/bpf_loader" }
-lua_loader = { path = "programs/native/lua_loader" }
+solana-program-native-noop = { path = "programs/native/noop" }
```

Comment:
Does 'program' really need to be in there?

Reply:
I guess not, I'll remove it

Reply:
I'm thinking `native` could go away too. Why would the choice of interpreter leak into the package name?